x-pack/solutions/security/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -62,6 +62,9 @@ export const SessionView = ({
   loadAlertDetails,
   canReadPolicyManagement,
   trackEvent,
+  openDetails,
```

Comment:
for comprehensibility, what do you think about adding a flyout or split mode reference here? something like `openDetailsInFlyout`

Reply:
good point, I'm changing it to `openDetailsInExpandableFlyout` to be even more specific 😆

Reply:
that was a nice suggestion 💯